webui/react/src/components/LogViewer/LogViewerFilters.settings.ts

```diff
@@ -10,48 +12,53 @@ export interface Settings {
   searchText?: string;
 }
 
-const config: SettingsConfig = {
-  settings: [
-    {
-      key: 'allocationId',
-      type: {
-        baseType: BaseType.String,
-        isArray: true,
-      },
+const config: SettingsConfig<Settings> = {
+  applicableRoutespace: 'log-viewer-filters',
+  settings: {
+    agentId: {
+      defaultValue: undefined,
```

Comment:
question: JSON doesnt support `undefined` value. How undefined data is stored in db?